src/controllers/signAccountOp/signAccountOp.ts

```diff
@@ -1189,18 +1203,32 @@ export class SignAccountOpController extends EventEmitter {
 
         if (usesPaymaster) {
           try {
-            const response = await this.#callRelayer(
-              `/v2/paymaster/${this.accountOp.networkId}/sign`,
-              'POST',
-              {
+            const santinelTimeoutErr = {}
```

Comment:
What does this mean `santinelTimeoutErr`

Reply:
This is how it was proposed by Ivo in estimate.ts and I just reused it. I agree it can be better